yt/frontends/ramses/particle_handlers.py

```diff
@@ -101,6 +105,15 @@ def read_header(self):
         pass
 
 
+_default_dtypes: dict[int, str] = {
+    1: "c",  # char
+    2: "h",  # short,
+    4: "f",  # float
+    8: "d",  # double
+    # 16: np.float128,  # double double precision
```

Comment:
should this be uncommented ?

Reply:
It should be removed, because the code path will not work (not supported by our reader at the moment.)